ledger-core/virtual/integration/delegated_finished_test.go

```diff
@@ -195,15 +195,12 @@ func TestVirtual_SendDelegatedFinished_IfPulseChanged_Without_SideEffect(t *test
 	rawWalletState := makeRawWalletState(t, testBalance)
 	msg := makeVStateReportEvent(server.GetPulse().PulseNumber, objectRef, stateID, rawWalletState)
 
-	server.WaitIdleConveyor()
```

Comment:
sure?

Reply:
yep, WaitIdleConveyor now in Server Init, and it is ready to serve